gr-soapy/grc/soapy_sdrplay_source.block.yml

```diff
@@ -76,7 +90,7 @@ templates:
   imports: from gnuradio import soapy
   make: |
       None
-      dev = 'driver=sdrplay'
+      dev = 'driver=sdrplay,agc_setpoint=-30'
```

Comment:
correct me if I'm wrong, but the `agc_setpoint` parameter above isn't connected to anything as this is hardcoded?